src/qt/notebook.cpp

```diff
@@ -178,14 +178,12 @@ bool wxNotebook::DeleteAllPages()
 
     // Block signals to not receive selection changed updates
     // which are sent by Qt after the selected page was deleted.
-    m_qtTabWidget->blockSignals(true);
+    wxQtEnsureSignalsBlocked blocker(m_qtTabWidget);
 
     // Pages will be deleted one by one in the base class.
     // There's no need to explicitly clear() the Qt control.
     bool deleted = wxNotebookBase::DeleteAllPages();
```

Comment:
Very minor, but this could be combined with the next line to just
```suggestion
    return wxNotebookBase::DeleteAllPages();
```
now.